packages/@aws-cdk/aws-ecs/lib/container-definition.ts

```diff
@@ -499,14 +566,6 @@ export interface HealthCheck {
   readonly timeout?: cdk.Duration;
 }
 
-function renderKV(env: { [key: string]: string }, keyName: string, valueName: string): any {
```

Comment:
Any reason this was changed?

Reply:
This is because `environment` was not using this function anymore with the previous implementation, will revert this.